mysql-test/mroonga/include/mroonga/check_version.inc

```diff
@@ -15,28 +15,32 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 
 --disable_query_log
-let $version_major_minor =
-  `SELECT CAST(SUBSTRING_INDEX(@@global.version, '.', 2) AS DECIMAL(4, 2))`;
+let $version_major =
+  `SELECT CAST(SUBSTRING_INDEX(@@global.version, '.', 1) AS UNSIGNED)`;
+let $version_minor =
```

Comment:
I think this is not elegant.
Is there other good way to get major/minor versions?